docs/src/main/asciidoc/migration-guide-1.x.adoc

```diff
@@ -0,0 +1,37 @@
+== Migration Guide from Spring Cloud GCP 1.x
+
+=== Maven Group ID Change
+Spring Cloud  link:https://spring.io/blog/2019/07/24/simplifying-the-spring-cloud-release-train[unbundled] Spring Cloud GCP and other IaaS providers from their release train.
+To use the newly unbundled libraries, change the group IDs in your `pom.xml` files:
+
+[source,xml]
+.pom.xml
+----
+<dependency>
+  <groupId>com.google.cloud</groupId> <!-- formerly org.springframework.cloud</groupId> -->
+  <artifactId>spring-cloud-gcp-starter-config</artifactId>
+</dependency>
+----
+
+=== Java Package Name Change
+All code in Spring Cloud GCP has been moved from `org.springframework.cloud.gcp` over to the `com.google.cloud.spring` package.
+
+=== Deprecated Items removed
+
+`GCPDatastoreProperties.emulatorHost`:: Use `host` instead
+`GcpPubSubHeaders.ACKNOWLEDGEMENT`:: Use `GcpPubSubHeaders.ORIGINAL_MESSAGE`, which is of type `BasicAcknowledgeablePubsubMessage`
+`SpannerQueryOptions.getQueryOptions`:: Use `getOptions()`
+`PubSubTemplate.subscribe(String, MessageReceiver)`:: Use `subscribe(String, Consumer<BasicAcknowledgeablePubsubMessage>)`
+`SpannerReadOptions.getReadOptions()`:: Use `getOptions`
+Stackdriver Logging::
+  `org.springframework.cloud.gcp.autoconfigure.logging` classes::: Use `com.google.cloud.spring.logging` from the `spring-cloud-gcp-logging` module.
+  Stackdriver Logback Appenders::: Use `com/google/cloud/spring/logging/logback-appender.xml` or `com/google/cloud/spring/logging/logback-json-appender.xml` from the `spring-cloud-gcp-logging` module.
```

Comment:
Do you think it would be more clear to say something like "Replace `org/springframework/cloud/gcp/autoconfigure/logging/logback-appender.xml` with `com/google/cloud/spring/logging/logback-appender.xml`" Or is it better to just state the new appender location?

Reply:
Done.